website/content/userguide/qemu.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "QEMU x86-64 virtual machine"
+menuTitle: "QEMU x86-64 virtual machine"
+weight: 70
+---
+
+In addition to running gokrazy on a Raspberry Pi or another ARM board, you can also run it in a QEMU x86-64 virtual
+machine. The following guide covers what's necessary to build and run a gokrazy image on QEMU. Your hypervisor may require additional configuration.
+
+## Before you begin
+
+If you don't already have Go installed, [install the latest version](https://golang.org/dl/). After that, install the `gok` command line tool:
+
+```bash
+go install github.com/gokrazy/tools/cmd/gok@main
+```
+
+## Building the image
+
+### Create a new gokrazy instance
+
+```bash
+gok new -i gok-vm
+```
+
+The `-i` flag specifies the name of the instance. In this case, we're using `gok-vm`, but you can use any name you like. The instance configuration will be created in your home directory under `~/gokrazy/gok-vm`.
+
+### Edit the instance configuration
+
+Open the instance configuration file in your favorite editor. The file is located at `~/gokrazy/gok-vm/config.json`. You'll need to make the following changes:
+
+{{< highlight json "hl_lines=3-4" >}}
+{
+  "Hostname": "gok-vm",
+  "KernelPackage": "github.com/rtr7/kernel",
+  "FirmwarePackage": "github.com/rtr7/kernel",
+  // ...
+}
+{{< /highlight >}}
+
+### Build the image
+
+The following command will build a single image file that contains the kernel, firmware, and root filesystem. The `--target_storage_bytes` flag must be specified to set the size of the image. The example below creates a 4GB image.
+
+```bash
+GOARCH=amd64 gok overwrite -i gok-vm --full gok-vm.img --target_storage_bytes 4294967296
+```
+
+The image will be written to the current directory as `gok-vm.img`.
+
+### Use the image with QEMU
+
+You can now use the image with QEMU. The following command will boot the image in QEMU with a serial console:
+
+```bash
+qemu-system-x86_64 -m 2048 -serial stdio -drive file=gok-vm.img,format=raw
```

Comment:
I typically add:

* `-machine accel=kvm` on Linux 
* `-smp 8` to make more than one core available (seems to be required, at least when I experimented with router7)